stargate/Dockerfile

```diff
@@ -5,8 +5,7 @@
 ARG BASE_IMAGE=ekidd/rust-musl-builder:latest
 FROM alpine/git:latest AS cloner
 
-RUN git clone https://github.com/apollographql/federation.git /src && \
-  git -C /src checkout jbaxleyiii/gateway-spike
+RUN git clone https://github.com/apollographql/federation.git /src
 
 # Our first FROM statement declares the build environment.
 FROM ${BASE_IMAGE} AS builder
```

Comment:
This only existed as a workaround for the time-being while we were based on this branch.   This is meant to be replaced with a [`COPY ../ /src`](https://docs.docker.com/engine/reference/builder/#copy) (inside the `builder` stage) so that this can be built from CI _from the current commit_.

Reply:
My Dockerfile foo isn't great to make sure I'm doing the right thing, I see what you're saying though, I hope it's ok but I'm going to merge this and maybe you can fix this later / we can do it tomorrow together?

Reply:
Hey @abernix , @trevor-scheer and I tried to address this comment, but couldn't quite figure it out quickly, we can't `COPY ../ .` (`/src` would be wrong) because it's a parent directory.
We tried some variations but failed at various stages. I had to stop for lunch and then do other more pressing stuff :)

Reply:
@Enrico2 @trevor-scheer  All good.  I can handle this!